config/module.config.php

```diff
@@ -2,11 +2,11 @@
 return [
     'service_manager' => [
         'invokables' => [
-            'Dash\Router\Http\Parser\ParserManager' => 'Dash\Router\Http\Parser\ParserManager',
-            'Dash\Router\Http\Route\RouteManager' => 'Dash\Router\Http\Route\RouteManager',
+            Dash\Router\Parser\ParserManager::class => Dash\Router\Parser\ParserManager::class,
```

Comment:
Well the point of use is having easier to read config. Most IDE automatically collapse the "use" lines anyway.